ports/argparse/vcpkg.json

```
@@ -1,7 +1,17 @@
 {
   "name": "argparse",
-  "version-date": "2021-03-04",
+  "version-date": "2021-09-14",
```

```suggestion
  "version": "2.2",
```

given that they release versions, we should use those versions